wordpress-mobile/WordPress-iOS
# Fixing Aztec crash on Xcode9

### Details:
This PR fixes a crash "at launch" in Aztec. Please, note that this PR must be merged after #7764 is, since the CocoaLumberjack upgrade is included here (since it's required for Xcode 9).

Ref.: [lazy var's new behavior](https://bugs.swift.org/browse/SR-5172).

Fixes #7374

### To test:
- Build (this branch) with Xcode 9
- Launch Aztec
- Verify it doesn't crash anymore! 

Needs review: @nheagy 
Thanks in advance Nate!

cc @diegoreymendez
